Bootcamp.ioc

```diff
@@ -155,4 +155,4 @@ VP_SYS_VS_Systick.Mode=SysTick
 VP_SYS_VS_Systick.Signal=SYS_VS_Systick
 VP_TIM16_VS_ClockSourceINT.Mode=Enable_Timer
 VP_TIM16_VS_ClockSourceINT.Signal=TIM16_VS_ClockSourceINT
-board=Bootcamp
+board=Bootcamp
```

Comment:
very nitpicky lmao, but there is a blank line missing at the end of this file, so changes are still being committed.

Reply:
I'm not too sure how to fix this with git commands lol

Reply:
https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

you can use git log to find the hash of the commit before you changed this file